### PR TITLE
EPMRPP-85652 || Width of API Key table is different from table

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/components/main/navigationTabs/navigationTabs.scss [hunk 1]
```diff
@@ -17,7 +17,7 @@
 .navigation-tabs {
   width: 100%;
   box-sizing: border-box;
-  padding: 5px 15px 15px 15px;
+  padding: 5px 45px 15px 45px;
```
Comment (Bam6ycha) [inline app/src/components/main/navigationTabs/navigationTabs.scss:20]: I think it is better not to change the padding directly in the general-purpose component.
It is used in several other places, and padding there 5px 15px 15px 15px;

File: app/src/components/main/navigationTabs/navigationTabs.scss [hunk 2]
```diff
@@ -27,7 +27,7 @@
   box-sizing: border-box;
   width: 100%;
   padding-top: 15px;
-  border-bottom: 1px solid $COLOR--gray-91;
+  border-bottom: 2px solid $COLOR--gray-91;
```
Comment (Bam6ycha) [inline app/src/components/main/navigationTabs/navigationTabs.scss:30]: I am not sure, that we need to change the width of this line.
https://www.figma.com/file/PxNk9h6CS9Y4Mxcl2fS9HF/RP5-Project-Level?type=design&node-id=1453-144839&mode=design&t=MVVjCbM1Y6Y6nktj-0
The same component with 1px border width
Discuss this with BA and UI/UX

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ export const ApiKeysBlock = ({ apiKeys }) => {
   const onRevokeClick = (key) => dispatch(showModalAction({ id: 'revokeApiKeyModal', data: key }));
 
   return (
-    <div className={cx('api-keys-block')}>
+    <div>
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx:64]: Could we replace it with Fragment?

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss [hunk 4]
```diff
@@ -14,11 +14,6 @@
  * limitations under the License.
  */
 
-.api-keys-block {
-  max-width: 860px;
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss:18]: According to the design, the width of each tab is 860 px
Are you really sure that we need to delete it here?
Mb we need to change the width of other tabs?
Could you please discuss it with BA?

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/src/components/main/navigationTabs/navigationTabs.scss (hunk 2 above)
Comment (BlazarQSO) [reply app/src/components/main/navigationTabs/navigationTabs.scss:30]: according to the design from the ticket:
https://www.figma.com/file/PxNk9h6CS9Y4Mxcl2fS9HF/RP5-Project-Level?type=design&node-id=7756-47207&mode=design&t=UU8oBhhWIdmc6L2H-0

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx (hunk 3 above)
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx]: Done.

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss (hunk 4 above)
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss:18]: 860px when the screen is 1320px
If the screen is larger then the width of the content also increases. (Victoria confirmed)

File: app/src/pages/inside/profilePage/profilePage.jsx [hunk 5]
```diff
@@ -117,9 +117,11 @@ export class ProfilePage extends Component {
               onChangeTab={this.props.dispatch}
             />
           </div>
-          <div className={cx('footer')}>
-            <LocalizationBlock />
-            <DeleteAccountBlock />
+          <div>
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/profilePage.jsx:120]: Why do we need this extra div wrapper here?

File: app/src/pages/inside/profilePage/profilePage.scss [hunk 6]
```diff
@@ -29,12 +29,16 @@
     .section-wrapper {
       padding: 0 30px;
     }
+    [class^="navigationTabs__navigation-tabs"] {
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/profilePage.scss:32]: I think that we can achieve the same result by adding NavigationTabs to the section-wrapper div
Screenshot 2023-10-03 at 10 56 42
And we do not change anything in CSS

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/profilePage/profilePage.scss (hunk 6 above)
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/profilePage.scss]: Done.

File: app/src/pages/inside/profilePage/profilePage.jsx (hunk 5 above)
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/profilePage.jsx]: Done.

**Review by Bam6ycha — APPROVED** [review]

**Pull request merged**